packages/caqti-async/caqti-async.0.9.0/opam

```diff
@@ -0,0 +1,18 @@
+opam-version: "1.2"
+name: "caqti-async"
+authors: ["Petter A. Urkedal"]
+maintainer: "paurkedal@gmail.com"
+homepage: "https://github.com/paurkedal/ocaml-caqti/"
+bug-reports: "https://github.com/paurkedal/ocaml-caqti/issues"
+dev-repo: "https://github.com/paurkedal/ocaml-caqti.git"
+license: "LGPL-3 with OCaml linking exception"
+
+build: [["jbuilder" "build" "-p" name "-j" jobs]]
+build-test: [["jbuilder" "runtest" "-p" name "-j" jobs]]
+
+depends: [
+  "async" {>= "v0.10.0"}
+  "core"
+  "caqti"
```

Comment:
This should depend on jbuilder.

Reply:
Right, build dependencies don't apply recursively.  Good catch.  Will add it for the other related packages, as well.